apps/web/src/components/templates/in-app-editor/ContentContainer.tsx

```diff
@@ -17,30 +18,11 @@ export function ContentContainer({
 }) {
   const theme = useMantineTheme();
   const ref = useRef<HTMLDivElement>(null);
-  const [visiblePlaceholder, setVisiblePlaceholder] = useState(!!value);
```

Comment:
There is no need for additional `useState`. This component is controlled meaning when value is passed it will re-render, so we can check that `value` and show/hide placeholder

Reply:
Great idea, love it!